src/backend/db/videos.js

```diff
@@ -67,7 +97,10 @@ export const videos = [
       "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
     creator: "Cryptoprenuer",
     videoImage:
-      "https://i.ytimg.com/vi/PHxVp0eEfPs/hqdefault.jpg?sqp=-oaymwEXCNACELwBSFryq4qpAwkIARUAAIhCGAE=&rs=AOn4CLCrL1MpVvOOedH4RloB8vXE3jSZig",
+      "https://i.ytimg.com/vi/Pvd1Y0OR2Nk/hqdefault.jpg?sqp=-oaymwEXCNACELwBSFryq4qpAwkIARUAAIhCGAE=&rs=AOn4CLCauoDvtkqaaSlaU5k8W1g-vA1hxQ",
+    views: "3",
+    videoCreated: "2",
+    videoUrl: "https://youtu.be/zj1X59IWLuQ",
   },
   {
     _id: uuid(),
```

Comment:
instead of uuid use youtube id it will help when you create single video page. It is as the last of URL if you use share feature on youtube

Reply:
![ss](https://user-images.githubusercontent.com/61417822/163412179-ac30b0e6-d0f0-4117-b26d-4955bbbb8006.png)

Reply:
Yes, it makes sense! I will do the refactor